Sources/Apollo/SelectionCondition+Evaluation.swift

```diff
@@ -0,0 +1,39 @@
+#if !COCOAPODS
+import ApolloAPI
+#endif
+import Foundation
+
+// MARK: Conditions - Or Group
+extension Selection.Conditions {
+  func evaluate(with variables: GraphQLOperation.Variables?) throws -> Bool {
+    for andGroup in value {
```

Comment:
`andGroup` is the correct name for the 'inner' arrays but this function is difficult to grok as it reads now - _we're evaluating the outer 'or' array with a quick exit should any of the inner arrays evaluates to true._ I wonder if there is a better way to express that?